### PR TITLE
add decode on client receiving message

### PR DESCRIPTION
**Pull request by jsateja**

### CODE REVIEW

File: echo_client.py [hunk 1]
```diff
@@ -14,5 +14,5 @@
 sock.sendall('Some message'.encode())
 data = sock.recv(1024)
 sock.close()
-print ("Received", repr(data))
+print ("Received", repr(data.decode()))
```
Comment (boogi) [inline echo_client.py]: Good to see that message is being decoded upon receiving by client.
I would remove also "repr" part and maybe set print to something like:
```suggestion
print ("Received: {}".format(data.decode()))
```

**Comment by jsateja** [comment]
Comment

**Review by boogi — COMMENTED** [review]
Another comment

**Comment by boogi** [comment]
and even another one

File: echo_client.py (hunk 1 above)
Comment (boogi) [inline echo_client.py]: Good change. I would also add different printing of message:
`print ("Received: {}".format(data.decode()))`

**Review by boogi — APPROVED** [review]
Seems like it can be merged now. Do it, please.

**Pull request merged**